queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -394,6 +373,25 @@ def reduce(self):
                         prettify(self.data))
             logger.info("Reduction job successfully complete")
 
+    def _new_reduction_data_path(self, path):
+        logger.info("_new_reduction_data_path argument: %s", path)
+        # if there is an 'overwrite' key/member with a None/False value
+        if 'overwrite' in self.data and not self.data["overwrite"]:
+            if os.path.isdir(path):           # if the given path already exists..
+                contents = os.listdir(path)
+                highest_vers = 0
```

Comment:
can we set this to `-1`? The minimum version in autoreduction is 0 so in the code below if you can't find a higher number it will increment on 0 so lowest version will be 1. but if we start at -1 then default will be 0.

Reply:
Can we chat about this? Just to confirm I'm on the same page